### DISCUSSION TITLE
Question on The The `scrollSeekConfiguration`

### DISCUSSION THREAD

**Discussion by zakuru** [petyosi/react-virtuoso#852] [General]
Hi, 
First amazing work on Virtuoso
I have an issue I want to fix but want to check first if is making sense to go that way
The documentation says The `scrollSeekConfiguration` property allows you to render a placeholder element instead of the actual item if the user scrolls too fast.
However would it be possible to limit the scroll speed and momementum to a certain Max value ?
If yes is there anyway to achieve it with the current implementation? If not, in case I am setting useWidowscroll to false can the behavior be implemented in the scroll logic of virtuoso (I would do it)
Thanks

**Comment by petyosi** [comment]
In short:

- limiting the scroll speed is not supported, regardless of the window/non-window mode
- the scrollSeekPlaceholder is not related to its hypothetical existence 
- the scrolling is native - interfering with its mechanics will detriment the user experience. Think of how something acts not the way you're used to.

**Reply by zakuru** [reply]
You are completely right that's what I had in mind as well 
Thanks a lot @petyosi
